webui/react/src/components/Spinner.module.scss

```diff
@@ -21,3 +29,10 @@
     animation: rotateCentered 1s linear infinite;
   }
 }
+.center-container {
```

Comment:
**nit:** `.centerContainer` so we can use `css.centerContainer` instead of `css['center-container']`